### ISSUE TITLE
[C++] Macro `#if defined MACRO` is not highlighted

### ISSUE THREAD

**Issue by evandrocoan** [sublimehq/Packages#675]
### [C++] Macro `#if defined MACRO` is not highlighted

``` cpp
    class Adapter : public Abstraction

    #if defined ASPECTO_MACRO
    , public Scenario
    #endif

    {
    public:
        Adapter();
    }

#if defined ASPECTO_MACRO
, public Scenario
#endif
```

On the vanilla install it is like this:

![image](https://cloud.githubusercontent.com/assets/5332158/19860379/9ff7e838-9f6f-11e6-8182-70af868fae35.png)

Here we note the correct highlight when it is defined outside the class declaration at the line 12.

Related threads: 

1. https://github.com/evandrocoan/SublimeTextStudio/issues/14 _ This whole macro body `#if defined ASPECTO_MACRO` is not highlighted on the Sublime Text C++ Syntax

**Comment by wbond** [comment]
This appears to be related to leading whitespace

**Comment by wbond** [comment]
This was fixed by f3e5ecc6f228df344a0edb423bdff2f07967e28e and will be part of the next dev build
